form_app/lib/src/autofill.dart

```diff
@@ -31,13 +57,21 @@ class _AutofillDemoState extends State<AutofillDemo> {
                   ...[
                     Text('This sample demonstrates autofill. '),
                     TextFormField(
+                      onFieldSubmitted: (vaue) {
+                        _lastname.requestFocus();
```

Comment:
If we're adding focus nodes to the other fields, it's probably a good idea to add one to this field as well, in order to complete the set. I wouldn't be surprised if it mattered for tab-order when building for the web, for example.